configuration/pom.xml

```diff
@@ -21,6 +21,13 @@
             <artifactId>validation-api</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.glassfish</groupId>
+            <artifactId>javax.el</artifactId>
+            <version>3.0.0</version>
+            <scope>runtime</scope>
+        </dependency>
```

Comment:
I am concerned about adding this dependency. `configuration` module is widely used and there is always a risk we will need to do dependency management cleanup because of this, also on downstream libraries.

This seems to be needed to produce a better error message when validation fails,
ie one including the validated value.
While I recognize the benefit, I think we should not add a new dependency for this.

Reply:
Sure, can do without it :)